charts/domain-server/templates/statefulset.yaml

```diff
@@ -92,6 +90,14 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+    volumes:
+    - name: private-key-volume
+      secret:
+        secretName: {{ include "domain-server.fullname" . }}
+        defaultMode: 0400
+        items:
+        - key: wallet_private_key
```

Comment:
Let's use the filename as the key name?

Reply:
https://github.com/aukilabs/terraform-live/pull/118